worker/remoterelations/remoterelations.go

```diff
@@ -170,7 +177,9 @@ func (w *Worker) handleApplicationChanges(applicationIds []string) error {
 		logger.Debugf("started watcher for remote application %q", name)
 		appWorker, err := newRemoteApplicationWorker(
 			relationsWatcher,
+			result.Result.ModelUUID,
```

Comment:
Nothing explicit. I find keyed fields useful mainly for when there are nil defaults. Otherwise it's just unnecessary boilerplate.